components/movementsensor/imuvectornav/imu.go

```diff
@@ -196,50 +191,44 @@ func newVectorNav(
 	if err != nil {
 		return nil, err
 	}
-	smp := uint(100)
-	if v.polling > 0 {
-		smp = uint(v.polling)
-	}
 
 	// compensate for acceleration bias due to misalignement
-	err = v.compensateAccelBias(ctx, smp)
+	err = v.compensateAccelBias(ctx, uint(smp))
 	if err != nil {
 		return nil, err
 	}
 	// compensate for constant DV bias in mesurament
-	err = v.compensateDVBias(ctx, smp)
+	err = v.compensateDVBias(ctx, uint(smp))
 	if err != nil {
 		return nil, err
 	}
 
-	// optionally start a polling goroutine
-	if pfreq > 0 {
-		logger.CDebugf(ctx, "vecnav: will pool at %d Hz", pfreq)
-		waitCh := make(chan struct{})
-		s := 1.0 / float64(pfreq)
-		v.workers = goutils.NewBackgroundStoppableWorkers(func(cancelCtx context.Context) {
-			timer := time.NewTicker(time.Duration(s * float64(time.Second)))
-			defer timer.Stop()
-			close(waitCh)
-			for {
-				select {
-				case <-cancelCtx.Done():
-					return
-				default:
-				}
-				select {
-				case <-cancelCtx.Done():
+	logger.CDebugf(ctx, "vecnav: will poll at %d Hz", smp)
+	waitCh := make(chan struct{})
+	s := 1.0 / float64(smp)
```

Comment:
What is s?

Reply:
seems to be the hertz version of the smp aka pollFreq. I can rename s to pollHertz. Thoughts?